packages/client/src/components/app/forms/RelationshipField.svelte

```diff
@@ -18,6 +18,7 @@
   export let filter
   export let datasourceType = "table"
   export let primaryDisplay
+  export let valueConversion
```

Comment:
Could we hide the details of this inside `BBReferenceField` instead of introducing it to the relationship field? By that I mean that we can pass in a custom `onChange` prop which does the conversion, and also pass in a custom `defaultValue` prop, which is already converted.

In my opinion it's just a little cleaner to contain this conversion logic inside the place that it's needed, the reference field, and leave the relationship field untouched.